tai64/Cargo.toml

```
@@ -17,10 +17,12 @@ maintenance = { status = "passively-maintained" }
 [dependencies]
 chrono = { version = "0.4", optional = true, default-features = false }
 serde = { version = "1", optional = true, default-features = false }
+zeroize = { version = "1.1.0", optional = true, default-features = false, features = ["zeroize_derive"] }
```

If you're not using the custom derive, remove the feature for it:
```suggestion
zeroize = { version = "1", optional = true, default-features = false }
```